.npmrc

```diff
@@ -1,2 +1,5 @@
-registry=http://registry.npmjs.org/
-engine-strict=true
+@cortexapps:registry=https://npm.pkg.github.com/
+//npm.pkg.github.com/:_authToken=${PACKAGES_REGISTRY_GITHUB_AUTH_TOKEN}
```

Comment:
Is there any other way of doing this? Or do we now require this env var to be setup just to yarn/npm install and run from now on

Reply:
@jdhayford because the `@cortexapps/birdseye` is needed to get pulled in, we need a token to access our internal github registry which we do via env variables. 

So yes you would need that token to get setup now.

Reply:
Got it. If this is the case and it is fully unavoidable, i.e. i cant `git commit` without this env var, we should likely add documentation about it in the readme and also be aware that the public cannot meaningfully interact with this repo

Reply:
@jdhayford unless we want to make it easier for cortex devs to contribute we could set it back to `GITHUB_PASSWORD` even though they say to not have `GITHUB_` prefixes.

In regards to public interacting we could proabbly figure that out after the fact, but also I don't know that there ever was that much contribution from the public?